pages/1.10/release-notes/1.10.6/index.md

```diff
@@ -45,11 +50,6 @@ DC/OS 1.10.0 includes many new capabilities for Operators and expands the collec
   
 Please try out the new features and updated data services. Provide any feedback through our support channel: <a href="https://support.mesosphere.com/">support.mesosphere.com</a>.
 
-### Contents
-- [New Features and Capabilities](#new-features)
-- [Breaking Changes](#breaking-changes)
-- [Known Issues and Limitations](#known-issues)
-
 # <a name="new-features"></a>New Features and Capabilities
 
 ## Apache Mesos 1.4 and Marathon 1.5 Integrated.
```

Comment:
I think this should be a `###` heading

Reply:
Also, instead of listing the versions of Mesos and Marathon that were integrated with 1.10.0 we should check for bumps and list all the integrated technology versions in 1.10.6. 

Remove this section and create a new section for 1.10.6 so that its clear what versions are integrated with this version.

Reply:
Done.